Dockerfile

```diff
@@ -20,6 +20,12 @@ RUN apt-get update \
  && chmod 777 /cache /config /media
 COPY --from=ffmpeg / /
 COPY --from=builder /jellyfin /jellyfin
+
+ARG JELLYFIN_WEB_VERSION=10.2.2
```

Comment:
the problem here is that we're trading a version set in submodule to version set in docker script... this would need to be updated at each release